### ISSUE TITLE
Transillumination of a mass shows high light transmission

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10767]
**Preferred term label:**
Transillumination of a mass shows high light transmission

**Synonyms**
Light transmitted through tissue

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ganglion cyst (hand or wrist)

**Your nano-attribution (ORCID)**

**Comment by MickeySegal** [comment]
A transillumination test either shows fluid or a solid mass that doesn't transmit light.  So just calling the test "positive" is not definitive.  So perhaps "Transillumination test shows high light transmission" would be the wording for a general term.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001424
- label: Transillumination test shows high light transmission
- parents: [Characteristic of mass (HP:0025758)](https://hpo.jax.org/app/browse/term/HP:0025758)
- pmid: PMID:20124823
- def: A transillumination test involves shining light through an area of the body or a mass. This term refers to a transillumination test that shows especially high (positive) transillumination, which may be observed when a mass or swelling contains clear fluid rather than solid tissue or blood.
- comment: Ganglion cysts may show high transillumination.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10767
- synonyms: Bright transillumination(NONE)
